fiware-region-sanity-tests/resources/userdata.test.snat

```diff
@@ -0,0 +1,26 @@
+#cloud-config
+
+# phone_home: if this dictionary is present, then the phone_home
+# cloud-config module will post specified data back to the given
+# url
+# default: none
+# phone_home:
+#  url: http://my.foo.bar/$INSTANCE/
+#  post: all
+#  tries: 10
+#
+# Phone Home service will be listening in: fi-health.lab.fiware.org:8081
+phone_home:
+ url: http://fi-health.lab.fiware.org:8081/
+ post: [ hostname, instance_id ]
+ tries: 10
+
```

Comment:
A minor detail: there is a variable PHONEHOME_PORT = 8081, but here the port is written directly as a number.

You can use a parameter an format.

I think this is not important and you can ignore this comment (it's OK to modify two files instead of one to change the port) but this change is also good to open a random port instead of a configurable one, that it is more robust to avoid cheating or attacks.